projects/tuist/fixtures/ios_app_with_actions/App/Project.swift

```diff
@@ -12,6 +12,7 @@ let project = Project(name: "App",
                                 .pre(tool: "/bin/echo", arguments: ["\"tuist\""], name: "Tuist", inputPaths: ["Sources/**/*.swift"]),
                                 .post(tool: "/bin/echo", arguments: ["rocks"], name: "Rocks"),
                                 .pre(path: "script.sh", name: "Run script"),
-                                .pre(script: "echo 'Hello World'", name: "Embedded script"),
+                                .post(script: "echo 'Hello World from install build'", name: "Embedded script install build", runForInstallBuildsOnly: true),
+                                .pre(script: "echo 'Hello World'", name: "Embedded script")
```

Comment:
One test still failing, as @kwridan mentioned I think there is a test that verifies this in a particular order

> sorry for the hassle, with the acceptance tests - one of them seems a flaky (being temporarily disabled via #2827), and looks like we had another that was verifying the first and last build phase script in that fixture, we'd nee to add the new example somewhere in the middle to not break those.
> 
> e.g.
> 
> actions: [
>     // Note there are acceptance tests verifying the first `pre` and last `post` action
>     // additions not part of the acceptance test should be added inbetween
>     .pre(tool: "/bin/echo", arguments: ["\"tuist\""], name: "Tuist", inputPaths: ["Sources/**/*.swift"]),
>     .post(script: "echo 'Hello World from install build'", name: "Embedded script install build", runForInstallBuildsOnly: true),
>     .post(tool: "/bin/echo", arguments: ["rocks"], name: "Rocks"),
>     .pre(path: "script.sh", name: "Run script"),
>     .pre(script: "echo 'Hello World'", name: "Embedded script"),
> ]

Reply:
The test that verifies this order is here: https://github.com/tuist/tuist/blob/main/projects/tuist/features/generate-4.feature#L9-L18 so we can either fix the test or fix the position here 👍🏼

Reply:
it's not in the first or last position anymore, don't know what to change. Maybe you can fix yourself or guide me to the pace or even make a PR on my fork? thx

Reply:
it also depends on whether the action is `.post` or `.pre`, I believe. Try to move [this](https://github.com/tuist/tuist/blob/57b478f907c1749711c8ae44993ec5b85c0a0523/projects/tuist/fixtures/ios_app_with_actions/App/Project.swift#L15) above [this](https://github.com/tuist/tuist/blob/57b478f907c1749711c8ae44993ec5b85c0a0523/projects/tuist/fixtures/ios_app_with_actions/App/Project.swift#L13). Note that you can test this locally as per [here](https://docs.tuist.io/contributors/testing-strategy)

Reply:
done, all tests are nor green... thx for the help